src/nvlink/worker_32bit.py

```
@@ -0,0 +1,377 @@
+
+    def close(self):
+        """Close NV-Link connection"""
+        try:
+            if self.nvlink is not None:
+                self.nvlink.NVClose()
+                self.nvlink = None
+
+            self.initialized = False
+            pythoncom.CoUninitialize()
+
+            return {"success": True}
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)